### DISCUSSION TITLE
how can i get the milvus version?

### DISCUSSION THREAD

**Discussion by huangxueliang** [milvus-io/milvus#37706] [Show and tell]
is there a command to get the version of milvus?

**Comment by yhmo** [comment]
```
client = MilvusClient(uri='http://localhost:19530')
print(client.get_server_version())
```

**Reply by huangxueliang** [reply]
i don't want to the python,because it nend to write a python file to run.
is the a command such as "select version()" in mysql to get the version?

**Reply by yhmo** [reply]
Can you execute "select version()" without a mysql client?

**Reply by huangxueliang** [reply]
it means i dont want to use the python scripts,because it nend to import some pkg,i want just only a command to get it.

**Reply by yhmo** [reply]
Currently milvus mainly provides the following client/sdk:

- pymilvu ---- python sdk
- java sdk
- go sdk
- nodejs sdk
- restful

All these sdks require a specific environment except the restful api. But the restful api doesn't have the get_server_version interface.
